### DISCUSSION TITLE
Need help with helm upgrade

### DISCUSSION THREAD

**Discussion by ljie-16** [helmfile/helmfile#231] [Questions] [answered]
I am trying to update the pod values.yaml para values via helm install. However, it showed Error: failed to download "nodeName/charts/podName/templates/values.yaml" (hint: running `helm repo update` may help).
The command i use is: helm upgrade namespace nodeName/charts/PodName/templates/values.yaml --set xxx.yyy=2
Thank you very much.

**Comment by yxxhero** [comment, marked as answer]
@ljie-16 Please create new issue in helm/helm
